proto/internal/temporal/server/api/persistence/v1/executions.proto

```diff
@@ -279,6 +279,11 @@ message WorkflowExecutionInfo {
     // Should be deprecated once the transition history is fully launched
     repeated VersionedTransition previous_transition_history = 100;
     VersionedTransition last_transition_history_break_point = 101;
+
+    // This is the execution time when the WorkflowExecutionStarted event was written.
+    // A workflow reset changes the workflow start time, but preserves this field .
+    google.protobuf.Timestamp original_execution_start_time = 102;
```

Comment:
I think we don't need this either. The describeWorkflowExecution api already loads the workflowStartedEvent, we can just get the event time as the OriginalStartTime

Reply:
yes, in this can there is no need to add it. I will reuse eventTime